recipes/openfx/all/conanfile.py

```diff
@@ -0,0 +1,93 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class openfx(ConanFile):
+    name = "openfx"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://openeffects.org"
+    description = "OpenFX image processing plug-in standard."
+    topics = ("image-processing", "standard")
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "fPIC": [True, False],
+        "shared": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+        "shared": False,
+    }
+    requires = ("opengl/system", "expat/2.4.8")
+    exports_sources = "CMakeLists.txt", "cmake/*", "symbols/*"
+
+    generators = "cmake"
+    _cmake = None
+
+    def source(self):
+        tools.get(
+            **self.conan_data["sources"][self.version],
+            destination="source_subfolder",
+            strip_root=True
+        )
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.configure()
+        return self._cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    @property
+    def _build_modules(self):
+        return [
+            os.path.join(
+                "lib", "cmake", "conan-official-{}-variables.cmake".format(self.name)
+            ),
+            os.path.join("lib", "cmake", "OpenFX.cmake"),
+        ]
+
+    def package(self):
+        cmake = self._configure_cmake()
+
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+
+        cmake.install()
+
+        self.copy("*.symbols", src="symbols", dst="lib/symbols")
+        self.copy("*.cmake", src="cmake", dst="lib/cmake")
```

Comment:
probably hooks will complain about it. No problem, we can add a rule to exclude your recipe.